px-map-behavior-vector-layer.es6.js

```diff
@@ -0,0 +1,119 @@
+(function() {
+  'use strict';
+
+  /****************************************************************************
+   * BEHAVIORS
+   ****************************************************************************/
+
+  /* Ensures the behavior namespace is created */
+  window.PxMapBehavior = (window.PxMapBehavior || {});
+
+  /**
+   *
+   * @polymerBehavior PxMapBehavior.VectorLayer
+   */
+  PxMapBehavior.VectorLayerImpl = {
+    properties: {
+      /**
+       * A string that will be contain the geojson:
+       *
+       * @type {String}
+       */
+      data: {
```

Comment:
Makes sense to me. I didn't know that Polymer would automatically parse it if it is a string, which is very handy.